### DISCUSSION TITLE
alias to disc image

### DISCUSSION THREAD

**Discussion by e-piping** [moneymanagerex/moneymanagerex#5455] [Q&A]
hi there,
i installed MMEX on a MAC with apple silicon, i followed the installation instructions, but i just get an alias in my applications folder to a disc image on the desktop. it works, i can run the app no problem, but the 16.0 installed correctly before. is there something i should be doing differently?

**Comment by whalley** [comment]
For the installation did you drag the application to the Applications folder? That is how to install.

Sounds as if you are dragging the disk image to the Applications folder.

**Comment by e-piping** [comment]
Hi Mark,
thanks for getting back to me. first off it’s all a bit confusing. the update wizard says the latest stable version is 1.6.3, when i click the link, the site shows 1.6.1 as the latest version available. 
secondly, i opened the disc image as you can see and tried to drag the mmex icon to my applications folder in my user, which produced an alias, however, if i drag the icon to the applications folder alias in the mmex installation window, it worked.
see attached screen capture.
which version is the latest?

Heath
￼

**Comment by whalley** [comment]
v1.6.3 is the latest. Dragging within the DMG window is the right thing to do.

Which website is showing v1.6.1 as the latest?

**Comment by e-piping** [comment]
It’s the source forge site. I just clicked on the link in the update wizard and that’s where it took me.HeathSent from my iPadOn Feb 3, 2023, at 07:40, Mark Whalley ***@***.***> wrote:﻿
v1.6.3 is the latest. Dragging within the DMG window is the right thing to do.
Which website is showing v1.6.1 as the latest?

**Reply by whalley** [reply]
What was the original version?.... Update wizard has been pointing at GitHub for some time now.

**Comment by e-piping** [comment]
1.6.0
Thank You,

Heath Lansdowne

 <http://www.heathlansdownedesign.ca/>
￼




250.361.5996
Modern, Efficient Home Design
heathlansdownedesign.ca <http://www.heathlansdownedesign.ca/>


￼

**Comment by e-piping** [comment]
can you send me the correct link please.

Thank You,

Heath Lansdowne

 <http://www.heathlansdownedesign.ca/>
￼




250.361.5996
Modern, Efficient Home Design
heathlansdownedesign.ca <http://www.heathlansdownedesign.ca/>


￼

**Reply by whalley** [reply]
Download latest from here: https://moneymanagerex.org/
